Library/SharedFunctions.swift

```diff
@@ -64,6 +64,23 @@ internal func reward(from backing: Backing, inProject project: Project) -> Rewar
     ?? Reward.noReward
 }
 
+/**
+ Returns a reward for a backing ID in a given project
+
+ - parameter backing: A backing ID
+ - parameter project: A project
+
+ - returns: A reward
+ */
+
+internal func reward(withId rewardId: Int, inProject project: Project) -> Reward {
```

Comment:
We also have this function in `SharedFunctions.swift`:
```
/**
 Returns a reward from a backing in a given project

 - parameter backing: A backing
 - parameter project: A project

 - returns: A reward
 */

internal func reward(from backing: Backing, inProject project: Project) -> Reward {
  let noRewardFromProject = project.rewards.first { $0.id == Reward.noReward.id }

  return backing.reward
    ?? project.rewards.first { $0.id == backing.rewardId }
    ?? noRewardFromProject
    ?? Reward.noReward
}
```
Seems pretty much identical, is it worthwhile to consolidate so we don't have to maintain this logic in two places?